src/app/(app)/layout.js

```diff
@@ -16,7 +17,7 @@ export default function Layout({ children }) {
   return (
     <Hint>
       <div className="flex min-h-screen flex-col">
-        <Header />
+        <Header socialLinks={<SocialLinksList className="text-primary-700 hover:text-primary-500" />} />
```

Comment:
Can we get social links here and pass them as a prop to the header and footer?

Reply:
I think we can do that. Though, how should "layout" page be revalidated?